vendor_skins/Evercast/CI/install/win/Install EBS.wxs

```diff
@@ -36,11 +36,13 @@
 				SourceFile=".\NDI 4 Runtime.exe"
 				Permanent="yes" />
 			<ExePackage
+				InstallCondition="NOT (VCRedist2015x64 = v14.25.28508)"
```

Comment:
Does it make sense to provide a "use at your own risk" alternative if this is not enough? I assume in extreme cases this issue could block someone from installing at all, is that correct?

Reply:
This won't stop EBS installation - the `Vital="no"` flag means EBS will continue if this fails.  Worst case (not known to exist, but possible) is that they'll still get the vcredist issue, at which point they'll have to install the redistributable manually.